src/pageLayouts/IliteHome/iliteHome.import.less

```diff
@@ -125,6 +127,7 @@ html {
       padding-left: 5vw;
```

Comment:
@jbiser361 The max-width here is correct, but the left/right padding should be on the parent so that the text itself can be 600px (currently it's 601 -  10vh)

Also, I don't recommend setting any left/right measurements to vh (it's height related) - try % instead